chatwoot/chatwoot
# feat: Show custom attributes in pre-chat form on new conversation

Fixes https://github.com/chatwoot/chatwoot/issues/6621

https://www.loom.com/share/bce7aae2c8334aaf99a47785b8182302

**How to test this PR?**
1. Add one conversation custom attribute.
2. Enable pre-chat form.
3. Enable the custom attribute created in step 1.
5. Enable "Allow messages after conversation resolved" from the inbox settings.
6. Create a conversation from the widget.
7. Resolve the conversation from the dashboard.
8. Click on the new conversation from the widget and check that the custom attribute is visible, not in the pre-chat form.
